Sources/Models/DataManager.swift

```diff
@@ -295,6 +299,9 @@ class DataManager: DataManagerProtocol {
         productApi.postProduct(product, rawParameters: nil, onSuccess: {
             DispatchQueue.main.async {
                 onSuccess()
+                if let barcode = product.barcode {
```

Comment:
@teolemon the view is reloaded for the current product, and only if the notification concerns the current displayed product